### PR TITLE
Issue and PR templates

### LINKED ISSUE #7: Add issue and pr templates
- [ ] Issue template: Bug 🐛
- [ ] PR template
- [ ] Spike Research Template

Issue and Spike templates should have acceptance criteria and need/value

### PR DESCRIPTION
**Pull request by TessFerrandez**
# PR for issue #7 

## What is being addressed

- Standardizing issue and PR documentation
- Closes #7 

## How is this addressed

- Add issue templates for bug, feature, user story, and research spike
- Add PR template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: 'bug'
+assignees: ''
+---
+Remember to label the issue with the appropriate area, and also assign it to yourself if you plan to work in the issue in the near future.
```
Comment (christoferlof) [inline .github/ISSUE_TEMPLATE/bug_report.md:8]:
```suggestion
Remember to label the issue with the appropriate area, and also assign it to yourself if you plan to work on the issue in the near future.
```

**Review by christoferlof — APPROVED** [review]

**Review by marrobi — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
```
Comment (tompaana) [inline .github/ISSUE_TEMPLATE/bug_report.md:14]:
```suggestion
**Steps to reproduce**
```

File: .github/ISSUE_TEMPLATE/research_spike.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+---
+name: Research Spike
```
Comment (tompaana) [inline .github/ISSUE_TEMPLATE/research_spike.md:2]:
```suggestion
name: Research spike
```

File: .github/ISSUE_TEMPLATE/user_story.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+---
+name: User Story
```
Comment (tompaana) [inline .github/ISSUE_TEMPLATE/user_story.md:2]:
```suggestion
name: User story
```

**Review by tompaana — APPROVED** [review]
Approving with suggestions

**Pull request merged**